### PR TITLE
DOCS-9207 adding hosts, ports, and certs info.

### PR DESCRIPTION
**Pull request by sleslie23**

### CODE REVIEW

**Review by sarjeet2013 — APPROVED** [review]

File: modules/ROOT/pages/install-self-managed.adoc [hunk 1]
```diff
@@ -11,37 +11,68 @@ Before installing Anypoint Runtime Fabric in a self-managed Kubernetes environme
...
-----
-====
-
+== Network Configuration
+
+Before installint or using Runtime Fabric, ensure that the following ports and hostnames are configured correctly.
+
+=== Port Configuration
+
+To install or run Runtime Fabric, ensure that you have configured the following ports on you Kubernetes installation:
```
Comment (sundippatel) [inline modules/ROOT/pages/install-self-managed.adoc]: typo: "you" -> "your"

File: modules/ROOT/pages/install-self-managed.adoc [hunk 2]
```diff
@@ -11,37 +11,68 @@ Before installing Anypoint Runtime Fabric in a self-managed Kubernetes environme
...
-If your Kubernetes configuration is not located in the `~/.kube/config` directory, set the `KUBECONFIG` environment variable before running `rtfctl`:
-
-----
-export KUBECONFIG=<path-to-kubeconfig>
-----
-====
-
+== Network Configuration
+
+Before installint or using Runtime Fabric, ensure that the following ports and hostnames are configured correctly.
```
Comment (sundippatel) [inline modules/ROOT/pages/install-self-managed.adoc]: typo: "installint" -> "installing"

File: modules/ROOT/pages/install-self-managed.adoc [hunk 3]
```diff
@@ -11,37 +11,68 @@ Before installing Anypoint Runtime Fabric in a self-managed Kubernetes environme
...
+
+=== Port Configuration
+
+To install or run Runtime Fabric, ensure that you have configured the following ports on you Kubernetes installation:
+
+[%header,cols="6*a"]
+.Port Configuration for Self-managed Kubernetes
+|===
+| Port | Layer 4 Protocol | Layer 5 Protocol | Source | Destination | Description
+| 443 | TCP | HTTPS | Internal | All nodes | Allow inbound requests to Mule runtime servers
```
Comment (sundippatel) [inline modules/ROOT/pages/install-self-managed.adoc]: Should the source be "external" or "internet"?

File: modules/ROOT/pages/install-self-managed.adoc [hunk 4]
```diff
@@ -11,37 +11,68 @@ Before installing Anypoint Runtime Fabric in a self-managed Kubernetes environme
...
+.Cerfificate configuration for self-managed kubernetes
+|===
+| Control Plane | Certificates
+| US control plane | transport-layer.prod.cloudhub.io + 
+configuration-resolver.prod.cloudhub.io
+| EU control plane | transport-layer.prod-eu.msap.io +
+configuration-resolver.prod-eu.msap.io
+|===
+
+== Download the rtfctl Command
```
Comment (sundippatel) [inline modules/ROOT/pages/install-self-managed.adoc]: suggest: "Command" -> "Utility"

File: modules/ROOT/pages/install-self-managed.adoc [hunk 5]
```diff
@@ -11,37 +11,68 @@ Before installing Anypoint Runtime Fabric in a self-managed Kubernetes environme
...
+| Control Plane | Certificates
+| US control plane | transport-layer.prod.cloudhub.io + 
+configuration-resolver.prod.cloudhub.io
+| EU control plane | transport-layer.prod-eu.msap.io +
+configuration-resolver.prod-eu.msap.io
+|===
+
+== Download the rtfctl Command 
+
+Runtime Fabric uses the `rtfctl` command for installation and management tasks.
```
Comment (sundippatel) [inline modules/ROOT/pages/install-self-managed.adoc]: suggest: "command" -> "command-line utility"

File: modules/ROOT/pages/install-self-managed.adoc [hunk 6]
```diff
@@ -71,6 +102,37 @@ curl -L https://devx.anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/lat
 sudo chmod +x rtfctl
 ----
 
+== Create a Runtime Fabric using Runtime Manager
+
```
Comment (sundippatel) [inline modules/ROOT/pages/install-self-managed.adoc:127]: We have at least two different personas performing actions on this doc: the *IT administrator* who is validating network requirements and installing RTF, and the *MuleSoft organizational administrator* who performs the actions within Anypoint Platform.

With that in mind, could we add a statement under this section to indicate these steps should be performed by a MuleSoft organization administrator?

File: modules/ROOT/pages/install-self-managed.adoc [hunk 7]
```diff
@@ -71,6 +102,37 @@ curl -L https://devx.anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/lat
...
+* Amazon Elastic Kubernetes Service
+* Azure Kubernetes Service
+
+. Click *Next*.
+. Review the *Support responsibility* disclaimer, then if you agree click *Continue*.
++
+Runtime Manager creates the Runtime Fabric and displays the Activation State page. This page displays the activation data used to install Runtime Fabric on a Kubernetes service. Copy this data to the clipboard for use in the next section. 
+
+
+== Install Runtime Fabric
```
Comment (sundippatel) [inline modules/ROOT/pages/install-self-managed.adoc]: From prior comment, could we add a statement under this section to indicate these steps should be performed by the IT administrator?

**Review by sundippatel — COMMENTED** [review]
Looks good overall; one suggestion to apply across the entire topic.

"command" -> "command-line utility"

File: modules/ROOT/pages/install-self-managed.adoc (hunk 1 above)
Comment (sleslie23) [reply modules/ROOT/pages/install-self-managed.adoc]: Fixed!

File: modules/ROOT/pages/install-self-managed.adoc (hunk 2 above)
Comment (sleslie23) [reply modules/ROOT/pages/install-self-managed.adoc]: Fixed

File: modules/ROOT/pages/install-self-managed.adoc (hunk 3 above)
Comment (sleslie23) [reply modules/ROOT/pages/install-self-managed.adoc]: Fixed. Verified everything else matches table in the Beta Google Doc.

File: modules/ROOT/pages/install-self-managed.adoc (hunk 4 above)
Comment (sleslie23) [reply modules/ROOT/pages/install-self-managed.adoc]: Changed.

File: modules/ROOT/pages/install-self-managed.adoc (hunk 5 above)
Comment (sleslie23) [reply modules/ROOT/pages/install-self-managed.adoc]: Changed.

File: modules/ROOT/pages/install-self-managed.adoc (hunk 6 above)
Comment (sleslie23) [reply modules/ROOT/pages/install-self-managed.adoc:127]: Added a note at the beginning of the topic:

- Runtime Fabric installation includes tasks that must be peformed by both IT administrators and MuleSoft organization administrators.

Also added a note at the beginning of each section:

- The procedures in this section should be performed by a MuleSoft organization administrator.

File: modules/ROOT/pages/install-self-managed.adoc (hunk 7 above)
Comment (sleslie23) [reply modules/ROOT/pages/install-self-managed.adoc]: Added a note to the beginning of the section:

The procedures in the section must be performed by an IT administrator.

File: modules/ROOT/pages/install-self-managed.adoc (hunk 4 above)
Comment (sleslie23) [reply modules/ROOT/pages/install-self-managed.adoc]: Globally changed command to command-line utility for rtfctl.

File: modules/ROOT/pages/install-self-managed.adoc [hunk 8]
```diff
@@ -65,12 +112,50 @@ curl -L https://devx.anypoint.mulesoft.com/runtimefabric/api/download/rtfctl-dar
...
+. From Anypoint Platform, select Runtime Manager.
+. Click *Runtime Fabrics*.
+. Click *Create Runtime Fabric*.
+. Enter the name of the new Runtime Fabric, then select one of the following options:
++
+* Amazon Elastic Kubernetes Service
+* Azure Kubernetes Service
+
+. Click *Next*.
+. Review the *Support responsibility* disclaimer, then if you agree click *Continue*.
```
Comment (sundippatel) [inline modules/ROOT/pages/install-self-managed.adoc:139]: "Continue" -> "Accept"

File: modules/ROOT/pages/install-self-managed.adoc [hunk 9]
```diff
@@ -65,12 +112,50 @@ curl -L https://devx.anypoint.mulesoft.com/runtimefabric/api/download/rtfctl-dar
...
+
+After creating a Runtime Fabric in Runtime Manager and obtaining the activation data, install Runtime Fabric into your Kubernetes service using the `rtfctl` command-line utility.
+
+If your Kubernetes configuration is not located in the `\~/.kube/config` directory, set the `KUBECONFIG` environment variable before running `rtfctl`:
+----
+export KUBECONFIG=<path-to-kubeconfig>
+----
+
+
 . Validate that you Kubernetes environment is read for installation:
```
Comment (sundippatel) [inline modules/ROOT/pages/install-self-managed.adoc:159]: you -> your

**Review by sundippatel — APPROVED** [review]

File: modules/ROOT/pages/install-self-managed.adoc (hunk 8 above)
Comment (sleslie23) [reply modules/ROOT/pages/install-self-managed.adoc:139]: Fixed.

File: modules/ROOT/pages/install-self-managed.adoc (hunk 9 above)
Comment (sleslie23) [reply modules/ROOT/pages/install-self-managed.adoc:159]: Fixed.

File: modules/ROOT/pages/install-self-managed.adoc (hunk 8 above)
Comment (sleslie23) [reply modules/ROOT/pages/install-self-managed.adoc:139]: Fixed.

File: modules/ROOT/pages/install-self-managed.adoc (hunk 9 above)
Comment (sleslie23) [reply modules/ROOT/pages/install-self-managed.adoc:159]: Fixed

**Pull request merged**